compute/admin_guide/install/install_kubernetes.adoc

```diff
@@ -251,7 +251,7 @@ Other options passed to _twistcli_ configure the chart.
 * Deploy Console and Defender with _helm install_ rather than _kubectl create_.
 
 [.procedure]
-. <<_download_twistlock,Download the Twistlock software>>.
+. xref:../welcome/releases.adoc#download[Download] the current recommended release.
```

Comment:
Hey @sgordon46 , I do not see this broken on prod: https://docs.paloaltonetworks.com/prisma/prisma-cloud/22-01/prisma-cloud-compute-edition-admin/install/install_kubernetes

Can you please confirm?